src/fsharp/symbols/Symbols.fs

```diff
@@ -2281,19 +2281,19 @@ type FSharpSymbolUse(g:TcGlobals, denv: DisplayEnv, symbol:FSharpSymbol, itemOcc
     member __.Symbol  = symbol
     member __.DisplayContext  = FSharpDisplayContext(fun _ -> denv)
     member x.IsDefinition = x.IsFromDefinition
-    member __.IsFromDefinition = (match itemOcc with ItemOccurence.Binding -> true | _ -> false)
-    member __.IsFromPattern = (match itemOcc with ItemOccurence.Pattern -> true | _ -> false)
-    member __.IsFromType = (match itemOcc with ItemOccurence.UseInType -> true | _ -> false)
-    member __.IsFromAttribute = (match itemOcc with ItemOccurence.UseInAttribute -> true | _ -> false)
-    member __.IsFromDispatchSlotImplementation = (match itemOcc with ItemOccurence.Implemented -> true | _ -> false)
+    member __.IsFromDefinition = itemOcc = ItemOccurence.Binding
```

Comment:
Minor stylistic nit - normally I'd put the second `=` in parentheses 

    member __.IsFromDefinition = (itemOcc = ItemOccurence.Binding)

Just one of those things arising from the multiple meanings of `=` in F# code